### ISSUE TITLE
Add ticket to re-run cases and set them to analyze

### ISSUE THREAD

**Issue by Vince-janv** [Clinical-Genomics/cg#1564]
## Description

When re-runs are ordered via the orderportal, by selecting an existing family, all that happens is that a ticket is opened. With the new tickets column in families (cases) we can now keep track of the ticket and while we're at it start the case as well.

## Suggested solution

- Append the new ticket number to the tickets column for the case/family
- Set the action of the family to `analyze`

## This can be closed when

Orders via the orderportal, selecting existing families are started automatically and have the new ticket added to them.

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by Vince-janv** [comment]
@fevac I think this might be a good issue for you to tackle! 👍

**Comment by fevac** [comment]
Fixed in: https://github.com/Clinical-Genomics/cg/pull/1565
